src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "ovn-kubernetes"
+---
+
+The **_ovn-kubernetes_** _networkplugin-syncer_ handler listens to Endpoints and
+Nodes and connects to the OVN databases (OVN NorthDB and SouthDB) where it
+maintains the necessary resources for Submariner:
+
+* **_submariner_router_** logical router, handles the communication to remote
+  clusters, and has a leg on the network which can talk to _ovn-k8s-gw0_ on
```

Comment:
nit: do we want to add a note about "_ovn-k8s-gw0_" interface?

Reply:
Based on the comments that I'm providing too much detail, I will skip :)